packages/react-core/src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -18,33 +27,34 @@ export interface BreadcrumbItemProps extends React.HTMLProps<HTMLLIElement> {
   target?: string;
   /** Sets the base component to render. Defaults to <a> */
   component?: React.ElementType;
+  /** A render function to render the component inside the breadcrumb item. */
+  render?: (props: BreadcrumbItemRenderArgs) => React.ReactNode;
 }
 
 export const BreadcrumbItem: React.FunctionComponent<BreadcrumbItemProps> = ({
   children = null,
-  className = '',
+  className: classNameProp = '',
   to = null,
   isActive = false,
   showDivider,
   target = null,
   component = 'a',
+  render = null,
   ...props
 }: BreadcrumbItemProps) => {
   const Component = component;
+  const ariaCurrent = isActive ? 'page' : undefined;
+  const className = css(styles.breadcrumbLink, isActive && styles.modifiers.current);
   return (
-    <li {...props} className={css(styles.breadcrumbItem, className)}>
+    <li {...props} className={css(styles.breadcrumbItem, classNameProp)}>
       {showDivider && (
         <span className={styles.breadcrumbItemDivider}>
           <AngleRightIcon />
         </span>
       )}
-      {to && (
-        <Component
-          href={to}
-          target={target}
-          className={css(styles.breadcrumbLink, isActive && styles.modifiers.current)}
-          aria-current={isActive ? 'page' : undefined}
-        >
+      {to && render && render({ className, ariaCurrent, to, isActive, target, showDivider, ...props })}
```

Comment:
thanks, this is almost good, but I don't think we need to require that the user pass `to` to use the render prop. All of those props (`to`, `isActive`, etc.) just get mirrored back to the user and doesn't provide more information. IMO the only useful props are className and ariaCurrent since we return something that the user didn't have to begin with.